src/skills-builder/skills-builder-modal/skillsBuilderModal.scss

```diff
@@ -3,3 +3,7 @@
     color: #ffffff;
   }
 }
+
+.med-min-height {
+  min-height: 5rem;
```

Comment:
We can use SCSS variables for these values as a best practice. I don't think it really has a huge effect here but these values could be changed to:
- `#ffffff` to `$white`
- `5rem` to `map-get($spacers, 6)`

I think we just missed the first one in a previous PR.

Reply:
Awesome, I'll add these suggestions!